.github/workflows/iroha2-dev-pr.yml

```diff
@@ -7,7 +7,7 @@ on:
       - '**.rs'
       - '**.json'
       - '**.toml'
-      - '.github/workflows/**.yml'
+      - '.github/workflows/iroha2-dev-pr.yml'
```

Comment:
I don't think we need to trigger tests by changing any workflow except for the testing workflow itself. What do you think @mversic

Reply:
agreed